tests/unit/test_param_expand.py

```diff
@@ -344,3 +347,122 @@ def test_parameter_graph_mixing_offset_and_conditional(self):
                     expected.replace(' ', '') in expanded,
                     f"Expected value {expected.replace(' ', '')} "
                     f"not in {expanded}")
+
+
+from typing import Any, Dict, Optional, Tuple
+
+
+class myParam():
+    def __init__(
+        self, raw_str,
+        parameter_values={}, templates={}, raises=None,
+        xfail=False,
+        id=None,
+        expect=None,
+    ):# -> None:
+        """Ease of reading wrapper for pytest.param
+
+        Args:
+            expect:
+                Output of expand_parent_params()
+            raw_str:
+                The parent_params input string.
+            parameter_values
+        """
+        self.xfail = xfail
```

Comment:
I don't think this xfail stuff is being used?

Reply:
Nope 

```suggestion
```